apps/openmw/mwmechanics/spellcasting.cpp

```diff
@@ -433,22 +426,28 @@ namespace MWMechanics
                                     "meshes\\" + reflectStatic->mModel, ESM::MagicEffect::Reflect, false, "");
                         reflectedEffects.mList.push_back(*effectIt);
                         magnitudeMult = 0;
+                        break;
```

Comment:
Specifically, effects further on the list continue to be checked even after one is absorbed/reflected.